### ISSUE TITLE
Accessibility (screen reader): ion-loading - the user gets no feedback about the loading process

### ISSUE THREAD

**Issue by galvanu** [ionic-team/ionic#11181]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[x] **2.x**
[ ] **3.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
ion-loading don't inform the user about the loading process and the displayed dialog when using accessibility voic assistant (screen reader).

**Expected behavior:**
The user should be notified about the new alert on the screen and about the loading indication.

**Steps to reproduce:**
1. Create a balnk page which shows ion-loading
2. run on device (android/ios)
3. enable screen reader/void assistant in accessibility settings
4. Open the app and navigate to the page
5. The loading alert is showing but the user has got no feedback about the loading indication

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Ionic Framework Version: 2.0.0
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.1.3
ios-deploy version: 1.9.1
ios-sim version: Not installed
OS: macOS Sierra
Node Version: v6.10.0
Xcode version: Xcode 8.2.1 Build version 8C1002
```

**Comment by jgw96** [comment]
Hello, thanks for opening an issue with us, we will look into this.

**Comment by trytuna** [comment]
Does this issue aim to get the audible loading indicator to work?

If you turn on VoiceOver on iOS and e.g. use Safari to load any website you will get audible feedback while loading the page. This is EXTREMELY useful for users with no sight.
